modules/admin-ui/src/main/webapp/scripts/shared/partials/modals/series-details.html

```diff
@@ -285,21 +285,33 @@ <h2 translate="EVENTS.SERIES.DETAILS.HEADER" translate-values="{{titleParams}}">
             <p class="tab-desc"></p>
             <div class="full-col">
                 <div class="obj quick-actions">
-                    <header>Theme</header>
+                  <header translate="CONFIGURATION.NAVIGATION.THEMES"><!-- Theme --></header>
                     <div class="obj-container padded">
-                        <select chosen pre-select-from="themes"
-                                allow-single-deselect="true"
-                                ng-disabled="(tab == 'theme') && !$root.userIs('ROLE_UI_SERIES_DETAILS_THEMES_EDIT')"
-                                data-width="'200px'"
-                                data-disable-search-threshold="8"
-                                ng-change="themeSave()"
-                                ng-model="theme.current"
-                                ng-options="id as name for (id, name) in themes"
-                                data-placeholder="{{ 'EVENTS.SERIES.NEW.THEME.LABEL' | translate }}"
-                                no-results-text="'{{ 'EVENTS.SERIES.NEW.THEME.EMPTY' | translate }}'"
-                                >
-                            <option value=""></option>
-                        </select>
+                        <ul>
+                            <li>
+                                <p translate="EVENTS.SERIES.NEW.THEME.DESCRIPTION.TEXT">
+                                    <!-- Theme Dialog Explanation Text -->
+                                </p>
+                                <p>
+                                    <!-- Dropdown with availabel themes -->
```

Comment:
~~availabel~~available

Also this comment is not in title case like the others. :wink: